promptsource/templates/squad_v2/templates.yaml

```diff
@@ -139,9 +145,12 @@ templates:
     metadata: !TemplateMetadata
```

Comment:
Can you fix some of the grammar in these options?

"From *the* passage, the topic is"
"What is *the* paragraph about?"

Also can you change the random to "choice".